keyboards/ph_arisu/readme.md

```diff
@@ -0,0 +1,16 @@
+# PH ARISU
+
+![PH_ARISU](https://www.dropbox.com/s/warl3bnuo49gxl1/PHArisu.jpg?raw=1)
```

Comment:
Please rehost this on Imgur.

Reply:
Copy Updated Image Host to Imgur

Reply:
https://github.com/qmk/qmk_firmware/pull/9612/commits/43812d6d0bb9b93703f3a657a327dbf221778338